source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/utils/wrappers/sb3.py

```diff
@@ -205,6 +205,9 @@ def seed(self, seed: int | None = None) -> list[int | None]:  # noqa: D102
 
     def reset(self) -> VecEnvObs:  # noqa: D102
         obs_dict, _ = self.env.reset()
+        # reset episodic information buffers
+        self._ep_rew_buf.zero_()
+        self._ep_len_buf.zero_()
```

Comment:
This reset function is usually only called once in the start, so it shouldn't affect the behavior. Can you share plots on how this change affects the training?

Reply:
I was trying to find the best hyperparameter for my RL model, so I created multiple agents using different hyperparameters in one shared Gym environment. Those models were trained one by one. The workflow was as follows:

```python
env = Sb3VecEnvWrapper(gym.make())
# test hyperparameter1
agent1 = PPO(policy_arch, env, **hyperparameter1)
agent1.learn()  # `env.reset()` is invoked within `agent.learn()` in SB3 package
# test hyperparameter2
agent2 = PPO(policy_arch, env, **hyperparameter2)
agent2.learn()
# test hyperparameter3
agent3 = PPO(policy_arch, env, **hyperparameter3)
agent3.learn()
...
env.close()
```

I'm sorry that I cannot provide the training curve plots as I've accidentally deleted them. As I recall, while agent1 exhibited a typical training curve, the subsequent agents displayed an unusual pattern, starting with excessively high values and then dropping rapidly. This fix successfully resolved the issue.